src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -29,51 +29,75 @@ import {
   styleUrls: ['./avatar.component.scss']
 })
 export class SkyAvatarComponent {
-  public get src(): SkyAvatarSrc {
-    return this._src;
-  }
 
+  /**
+   * Indicates whether users can change the image. To select a different image,
```

Comment:
In the existing docs `canChange` is marked as required, but that doesn't make sense given that it's a `boolean` with a default value. But just to make sure, we are good with with losing the required label, right?

Reply:
That's correct - it is not required. It should not have the label anymore.